apps/ui/src/components/SpaceTreasury.vue

```diff
@@ -120,6 +123,18 @@ async function addTx(tx: Transaction) {
 onMounted(() => {
   if (!treasury.value) return;
 
+  if (
+    !route.params.tab ||
+    !['tokens', 'nfts'].includes(route.params.tab as string)
+  ) {
+    return router.replace({
+      name: 'space-treasury',
+      params: {
+        tab: 'tokens'
+      }
+    });
+  }
+
```

Comment:
Is this expected that if I go to https://snapshot.box/#/s:fabien.eth/treasury I get redirected to /1/tokens
but if I go to https://snapshot.box/#/s:fabien.eth/treasury/1 I go redirected to /tokens (without treasury id)?

Reply:
@ChaituVR

Reply:
I think it's not because then the URL is broken (next refresh will show you `Treasury not found.`)

Reply:
Ahh yes, looks like I missed it in refactoring, will create a separate PR